radis/tools/database.py

```diff
@@ -1686,7 +1688,7 @@ def plot_cond(self, cond_x, cond_y, z_value=None, nfig=None):
 
         # Overlay color
         if z_value is not None:
-            assert (len(z_value)) == len(self.df)
+            assert (len(self.df[z_value])) == len(self.df)
 
             z = np.array(z_value) ** 0.5  # because the lower the better
```

Comment:
@erwanp I believe the above line `assert (len(z_value)) == len(self.df)` needs to be changed into `assert (len(self.df[z_value])) == len(self.df)` because z_value is just the column name.

Also the above below gives error:
 ```
   z = np.array(z_value) ** 0.5  # because the lower the better

TypeError: ufunc 'power' not supported for the input types, and the inputs could not be safely coerced to any supported types according to the casting rule ''safe
```

Reply:
I looks like that z_value could be used as an external array. What about : 
- if z_value is a `str` ; and a column name, then use `z_value = self.df[z_value]`
- else let it as it is (so the user can still use an external array of any kind) 

```np.array(z_value) ** 0.5```
looks like something added for specific (old) purposes, it could be removed.

Reply:
Plotted: `db_new.plot_cond('Tgas', 'wstep', 'wstep')` where wstep value was decreasing, got the below plot-

![Figure_1](https://user-images.githubusercontent.com/44584067/125667081-7e93443e-1e80-4d12-bcdd-84655bf9d417.png)

I believe the color of the point should be different instead of the whole area?

Reply:
I think I use it to plot the residuals in Fitroom, if you had a look at it. So the area is what we want. Let's use it : maybe compute  small matrix of Tgas and wstep (15-20 spectra max, but larger ranges, like up to 3000 K to show some differences) , and plot the calculation time as the Z_label? 

It's just to inspire people, no need to spend much time here :)

Reply:
Added a better example where I'm halving the wstep at every iteration. Now the difference is clearer and we can infer that Violent -> Minimum calc time, yellow -> Maximum Calc Time. Also, the max time taken is `0.17s` and overall takes around `0. 40` seconds to fully execute it so it's not expensive.

![Screenshot from 2021-07-15 03-49-33](https://user-images.githubusercontent.com/44584067/125700476-2004b16a-9921-4903-b400-8b440793d337.png)